stable/goldilocks/ci/test-values.yaml

```diff
@@ -31,6 +31,8 @@ controller:
 dashboard:
   basePath: goldilocks
   ingress:
+    extraClusterRoleBindings:
+      - view
```

Comment:
I don't think that's the right place for this. Shouldn't this go under `controller.rbac`?